app/src/main/java/cm/aptoide/pt/AptoideApplication.java

```diff
@@ -350,6 +354,13 @@ public LeakTool getLeakTool() {
     aptoideDownloadManager.start();
   }
 
+  private void initializeMopub() {
+    SdkConfiguration sdkConfiguration =
+        new SdkConfiguration.Builder(BuildConfig.MOPUB_BANNER_90_HOME_PLACEMENT_ID_T12).build();
+    MoPub.initializeSdk(this, sdkConfiguration, () -> {
```

Comment:
'this' and BuildConfig.MOPUB_BANNER_90_HOME_PLACEMENT_ID_T12 should be extracted to parameter to have the exact same initialization style as the initializeFlurry method.

If we want to ignore the initializeSdk completion, passing a null would be better in terms of performance - also because the listener parameter is marked as @ Nullable

Reply:
fixed